bazel/third_party/glog/0001-improvement-copts.patch

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
What is this file used for? Can we delete it?

Reply:
Sure, will do.